packages/block-editor/src/components/block-pattern-picker/layout-setup-toolbar.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import {
+	chevronRight,
+	chevronLeft,
+	grid,
+	stretchFullWidth,
+} from '@wordpress/icons';
+
+function LayoutSetupToolbar( {
+	viewMode,
+	setViewMode,
+	handlePrevious,
+	handleNext,
+	activeSlide,
+	totalSlides,
+	onBlockPatternSelect,
+	onStartBlank,
+} ) {
+	const isSingleView = viewMode === 'single';
+	const navigation = (
+		<div className="block-setup-navigation">
+			<>
+				<Button
+					icon={ chevronLeft }
+					label={ __( 'Previous block pattern' ) }
+					onClick={ handlePrevious }
+					disabled={ activeSlide === 0 }
+				/>
+				<Button
+					icon={ chevronRight }
+					label={ __( 'Next block pattern' ) }
```

Comment:
nit: I think we should avoid "block pattern" and just call them "next pattern" in UI labels